viya_ark_library/k8s/test_impl/response_data/resources_configmaps.json

```diff
@@ -62,5 +62,281 @@
+            }
+        },
+        {
+            "apiVersion": "v1",
+            "data": {
+                "SAS_ALLOW_ADMIN_SCRIPTS": "false",
+                "SAS_MULTI_TENANCY_ENABLED": "false",
+                "SAS_SERVICES_URL": "https://k8s-master-node.test.sas.com:443",
+                "SAS_URL_SERVICE_SCHEME": "https",
+                "SAS_URL_SERVICE_TEMPLATE": "https://@k8s.service.name@"
+            },
+            "kind": "ConfigMap",
+            "metadata": {
+                "annotations": {
+                    "kubectl.kubernetes.io/last-applied-configuration": ""
+                },
+                "creationTimestamp": "2021-08-26T16:06:32Z",
+                "labels": {
+                    "sas.com/admin": "cluster-local",
+                    "sas.com/deployment": "sas-viya"
+                },
+                "managedFields": [
+                    {
+                        "apiVersion": "v1",
+                        "fieldsType": "FieldsV1",
+                        "fieldsV1": {},
+                        "manager": "kubectl",
+                        "operation": "Update",
+                        "time": "2021-08-26T16:06:32Z"
+                    }
+                ],
+                "name": "sas-shared-config-dh4f2c4mk9",
+                "namespace": "test",
+                "ownerReferences": [
+                    {
+                        "apiVersion": "orchestration.sas.com/v1alpha1",
+                        "blockOwnerDeletion": true,
+                        "controller": true,
+                        "kind": "SASDeployment",
+                        "name": "sas-viya",
+                        "uid": "396aebe9-5080-4117-addb-f51ec1c254d5"
+                    }
+                ],
+                "resourceVersion": "12975367",
+                "selfLink": "/api/v1/namespaces/test/configmaps/sas-shared-config-dh4f2c4mk9",
+                "uid": "7a894053-f8f4-4bb1-861c-450f9ff09704"
+            }
+        },
+        {
+            "apiVersion": "v1",
+            "data": {
+                "SAS_RESTORE_JOB_DU_NAME": "sas-restore-job"
+            },
+            "kind": "ConfigMap",
+            "metadata": {
+                "annotations": {
+                    "kubectl.kubernetes.io/last-applied-configuration": ""
+                },
+                "creationTimestamp": "2021-08-26T16:06:32Z",
+                "labels": {
+                    "sas.com/admin": "cluster-local",
+                    "sas.com/deployment": "sas-viya"
+                },
+                "managedFields": [
+                    {
+                        "apiVersion": "v1",
+                        "fieldsType": "FieldsV1",
+                        "fieldsV1": {},
+                        "manager": "kubectl",
+                        "operation": "Update",
+                        "time": "2021-08-26T16:06:32Z"
+                    }
+                ],
```

Comment:
You can remove managedFields blocks to reduce file size.

Reply:
removed.